Create ui-c8y-1019-16-0-added-zip-argument-when-building-an-application-5452-graft-release-cd.md (#1859)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-16-0-added-zip-argument-when-building-an-application-5452-graft-release-cd.md

---
date: ""

> title: Zip option can now be provided when building an application. 

Reviewer: ```suggestion
title: ZIP option now available when building an application 
```

> title: Added --zip argument when building an application. (#5452) [GRAFT][release/cd] (#5923)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Add --zip argument for building applications
```

> title: Added --zip argument when building an application. (#5452) [GRAFT][release/cd] (#5923)

Reviewer 2: ```suggestion
title: Zip option can now be provided when building an application. 
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57342

> version: 1019.16.0

Reviewer: ```suggestion
version: 10.19.16.0
```

---

> Added --zip argument when building an application. (#5452) [GRAFT][release/cd] (#5923)

Author: The Claude AI suggested to use the following body:
```suggestion
To simplify the deployment of applications, a new --zip argument has been added to the application build command. When specified, the build process will now generate a ZIP archive containing the application and its dependencies in addition to the regular build artifacts. This allows users to easily package and distribute their applications in a single file. The change is fully backwards compatible and does not impact existing application builds or deployments.
```

> Added --zip argument when building an application. (#5452) [GRAFT][release/cd] (#5923)

Reviewer 2: ```suggestion
It is now possible to build an application with the --zip argument which now generate a ZIP archive containing the application and its dependencies in addition to the regular build artifacts.
```

> It is now possible to build an application with the --zip argument which now generate a ZIP archive containing the application and its dependencies in addition to the regular build artifacts.

Reviewer: ```suggestion
It is now possible to build an application with the --zip argument. This will generate a ZIP archive which contains the application and its dependencies in addition to the regular build artifacts.
```
